plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -132,17 +157,19 @@ When a user deactivates or uninstalls your plugin, it’s good practice to clean
 
 In most cases, plugin data is removed during uninstallation because it indicates the user no longer needs the plugin. As you learned in Module 1, you can use the `register_uninstall_hook()` function to do this.
 
-You can execute the SQL query to delete the table by using the `$wpdb` global variable, which we'll learn more about in the next lesson.
+You can execute the SQL query to delete the table by using the `$wpdb` global variable, which you'll learn more about in a future lesson.
 
 ```php
-register_uninstall_hook( __FILE__, 'my_plugin_delete_table' );
-function my_plugin_delete_table() {
+register_uninstall_hook( __FILE__, 'wp_learn_custom_table_delete_submissions_table' );
+function wp_learn_custom_table_delete_submissions_table() {
 	global $wpdb;
-	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}my_plugin_table" );
-	delete_option( 'my_plugin_db_version' );
+	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}submissions" );
+	delete_option( 'wp_learn_custom_table_db_version' );
 }
 ```
 
 ## Conclusion
 
-Now that we know how to create, update, and delete a custom table in the WordPress database, let's see how to read and write data within the table. We'll cover these concepts in the next lesson, "Interacting with Custom Tables".
+Now you know how to create, update, and delete a custom table in the WordPress database.
```

Comment:
This falls flat to me. The next paragraph about "more information" doesn't flow well because we just said that I now know how to do all these things, so I don't know what more information I could need. Is there a better transition/flow we could do for the conclusion here? 🤔

Reply:
Let me noodle on this a bit more.

Reply:
True, I think it makes sense to simply remove it. The sentence before this section transitions to this conclusion anyways by saying "which you'll learn more about in a future lesson." That's probably simple enough and this section can just say where to find more information like you have.